src/views/index.njk

```diff
@@ -1,21 +1,153 @@
+                </div>
+            </div>
+        </div>
+    </header>
 
-    {% include "_network_stats.njk" %}
-    {% include "_node_list.njk" %}
-    {% include "_info_the_federation.njk" %}
-    {% include "_info_site.njk" %}
-
-    <p class="text-center">
-        built with &lt;3 by <a href="https://jasonrobinson.me">jason robinson</a> and <a href="https://github.com/jaywink/the-federation.info/graphs/contributors">contributors</a>
-    </p>
+    <section class="tile">
+        <header>
+            <h2>What is The Federation?</h2>
+        </header>
+        <div>
+            <div class="flex">
+                <div class="col2">
+                    <p>
+                        The Federation refers to a global social network composed by nodes talking together.
+                        Each of them is an installation of a software which supports the <a href="https://github.com/SuperTux88/diaspora_federation">diaspora* protocol</a>.
+                        This website lists all the registered servers among the world to draw statistics. You know, we love numbers and numbers love us.
+                    </p>
+                    <div class="center">
+                        <a href="/global-stats.html" class="btn btn-primary">Visit network evolution</a>
+                    </div>
+                </div>
+                <div class="col2">
+                    <ul>
+                        <li>Nodes: <strong>{{ data.totals.pod_count }}</strong></li>
+                        <li>Users: <strong>{{ data.totals.total_users }}</strong></li>
+                        <li>Last 6 months active users: <strong>{{ data.totals.active_users_halfyear }}</strong></li>
+                        <li>Last month active users: <strong>{{ data.totals.active_users_monthly }}</strong></li>
+                        <li>Posts: <strong>{{ data.totals.local_posts }}</strong></li>
+                        <li>Comments: <strong>{{ data.totals.local_comments }}</strong></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="tile">
+        <header>
+            <h2>diaspora* - Friendica - Redmatrix - Hubzilla</h2>
+            <h3>Discover the projects which are composing the federation</h3>
+        </header>
+        <div>
+            <table>
+                <thead>
+                    <tr>
+                        <th>Project</th>
+                        <th><a href="/diaspora.html">diaspora*</a></th>
+                        <th><a href="/friendica.html">Friendica</a></th>
+                        <th><a href="/redmatrix.html">Redmatrix</a></th>
+                        <th><a href="/hubzilla.html">Hubzilla</a></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <th>First release date</th>
+                        <td>11-2010</td>
+                        <td>04-2011</td>
```

Comment:
I checked github and the first tag is [April 2011](https://github.com/friendica/friendica/releases?after=2.35) but it's a 2.1 so I guess the first release was before that.